code/datums/dog_fashion.dm

```diff
@@ -211,3 +211,15 @@
 
 /datum/dog_fashion/head/blumpkin/lit
 	obj_icon_state = "blumpkin1"
+
+/datum/dog_fashion/head/butter
+	name = "Butter REAL_NAME"
+	desc = "NAME. Dogname with the butter. NAME. Dogname with a butter on 'em."
+	obj_icon_state = "butter"
+	speak = list() //they're very patient and focused on holding the butter on 'em
+	emote_see = list("shakes a little.", "looks around.")
+	emote_hear = list("licks a trickle of the butter up.")
+
+/datum/dog_fashion/head/butter/New(mob/M)
+	desc = "NAME. [M.real_name] with the butter. NAME. [M.real_name] with a butter on 'em."
```

Comment:
what does NAME mean

Reply:
it gets replaced with the dog's full name, e. g. _Butter Mr. Woofsworth the IV, Esq._
https://github.com/tgstation/tgstation/blob/b488a846a1b469803bbc2cfca7b5621cfd4559ac/code/datums/dog_fashion.dm#L16-L18

Reply:
i'm going to change the placeholders so they won't clash with each other and add the REAL_NAME one to the desc as well to use here and in another item